Controller/Activation/ActivationController.php

```diff
@@ -151,15 +154,11 @@ public function changePasswordAction(Request $request, $token)
         );
     }
 
-    /**
-     * @param $token
-     * @return ActivableInterface|null
-     */
-    private function tryFindUserByActivationToken($token)
+    private function tryFindUserByActivationToken(string $token): ActivableInterface
     {
         $user = $this->userRepository->findUserByActivationToken($token);
 
-        if (!($user instanceof ActivableInterface)) {
```

Comment:
Not sure if such changes should be a part of this PR.

Reply:
`UserRepositoryInterface::findUserByActivationToken()` now returns always instance of ActivableInterface, so this check became redundant